app/src/main/java/com/github/libretube/ui/fragments/PlaylistFragment.kt

```diff
@@ -136,7 +136,8 @@ class PlaylistFragment : DynamicLayoutManagerFragment() {
             nextPage = response.nextpage
             playlistName = response.name
             isLoading = false
-            ImageHelper.loadImage(response.thumbnailUrl, binding.thumbnail)
+            if (!response.thumbnailUrl.isNullOrEmpty())
+                ImageHelper.loadImage(response.thumbnailUrl, binding.thumbnail)
```

Comment:
The `ImageHelper#loadImage` function already takes care of that to avoid too much code repetition in url validation.

Reply:
To be honest, I did not delve into the work of this function and everything related to it, but I hid it with a comment, and the problem with the layout disappeared by itself. It is possible that the solution is far from perfect, but it is the simplest, as it seemed to me.

Reply:
https://github.com/libre-tube/LibreTube/blob/a3245d814f0dd0c0847a1e12890333142562b3e1/app/src/main/java/com/github/libretube/helpers/ImageHelper.kt#L70

Reply:
This change doesn't have any impact.